sigma9.css

```diff
@@ -7,15 +7,17 @@
 
 @charset "utf-8";
 
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
Is this the canonical URL for Inter?

Reply:
This is the import that they suggest here: https://rsms.me/inter/#usage